### PR TITLE
Updated version of 352

### PR DESCRIPTION
**Pull request by eshaben**
### Description/Original PRs

This PR adds the translated content from #352. It also adds in some missing content (in English) that still needs to be translated.

I had to do it this way because #352 was opened from a fork so I couldn't push commits to that PR.

### CODE REVIEW

File: .snippets/text/node-operators/networks/client-changes.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+!!! note
```
Comment (eshaben) [inline .snippets/text/node-operators/networks/client-changes.md:1]: @wuzhong-papermoon - can you please translate this file?

File: builders/interoperability/xcm/xcm-sdk/v1/xcm-sdk.md [hunk 2]
```diff
@@ -98,11 +112,102 @@ const ethersSigner = new ethers.Wallet(privateKey, provider);
...
+    const account = privateKeyToAccount(privateKey);
+
+    const evmSigner = createWalletClient({
+      account,
+      chain: moonbaseAlpha,
+      transport: http(),
+    });
+    ```
+
+If you want to pass in a browser extension wallet to viem, you can use the following code:
```
Comment (eshaben) [inline builders/interoperability/xcm/xcm-sdk/v1/xcm-sdk.md:175]: @wuzhong-papermoon - can you please translate this line?

File: .snippets/text/node-operators/client-changes.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+!!! note
```
Comment (eshaben) [inline .snippets/text/node-operators/client-changes.md:1]: @wuzhong-papermoon - can you please translate this file?

File: .snippets/text/node-operators/client-changes.md
Comment (wuzhong-papermoon) [inline .snippets/text/node-operators/client-changes.md:1]: !!! 注意事项  
     在客户端v0.30.0之前的版本中，`--rpc-port`参数被用作定义HTTP连接端口，`--ws-port`参数被用作定义WS连接端口。在客户端v0.30.0之后的版本这两个参数被合并在一起，原本`--ws-port`的默认端口`9944`被HTTP与WS连接共同使用。这个端口的最大连接数在源码中的硬编码为100，这个数字可以被`--ws-max-connections`参数覆盖。

     从客户端v0.30.0之后，`--ws-port`与`--ws-max-connections`这两个参数已被弃用，被`--rpc-port`与`--rpc-max-connections`替代。默认端口仍为`9944`，默认最大连接数仍为100.

**Review by wuzhong-papermoon — APPROVED** [review]

**Pull request merged**